getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Events"
+weight: 20
+---
+
+# Events
+
+
+### Introduction
+
+Events are portions of data which can be sent to a chat ([send_event](../../agent-api/client-server#send-event)). There are different types of events:
+
+ - [message](#message)
+ - [annotation](#annotation)
+ - [filled-form](#filled-form)
+ - [system message](#system-message)
+ - [file](#file)
+ - [custom event](#custom-event)
+
+
+[Incoming event](../../agent-api/server-client#incoming-event) push will inform you about events sent to a chat (on both, agent and customer side).
+
+
+### Common event fields
+
+All events have some common fields, we will describe them here to avoid duplication:
+
+| name        | puprose                                        | filled by | type   |
+|-------------|------------------------------------------------|-----------|--------|
+| `id`        | id of the event                                | server    | string |
+| `custom_id` | is an ID that you can set for your own purpose | client    | string |
+| `order`     | event with lower `order` are hronologicaly before | server | int    |
+| `author_id` | this is id of sender of the event              | server    | string |
+| `text`      | this is payload that you can send as event (in example message content) | client | string |
+| `recipients` | posibble values are `"all"`(default) and `"agents"` | client | string |
+| `properties` | event properties, more about this in future properties section | client | properties object |
+
+
+### Message
+
+Message is the most common and self-explanatory event type. It allows sending some text to other chat users. The data format looks like this:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",  // generated by server side
+  "custom_id": "12345-bhdsa",
+  "order": 1,  // generated by server side
+  "type": "message",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,  // generated by server side
+  "text": "hello there",
+  "recipients": "all",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+
+
+### Annotation
+
+Adds annotation to last thread. It is important to mention that sending annotation cannot start a new thread (even if there is no active thread in a chat), it always goes to latest thread that alredy exist.
```

Comment:
Adds annotation to last thread. It is important to mention that sending annotation cannot start a new thread (even if there is no active thread in a chat), it always goes to latest thread that alredy exist. - >Adds an annotation to the last thread. Keep in mind that sending an annotation cannot start a new thread (even if there is no active thread in a chat). It always goes to the latest thread that already exists.